lib/firmware/fsm_msg_tendermint.h

```diff
@@ -154,7 +154,10 @@ void fsm_msgTendermintMsgAck(const TendermintMsgAck *msg) {
     case OutputAddressType_TRANSFER:
     default: {
       char amount_str[32];
-      bn_format_uint64(msg->send.amount, NULL, msg->denom, 6, 0, false, amount_str,
+      char suffix[sizeof(coin->coin_shortcut) + 1]; // sizeof(coin->coin_shortcut) includes space for the terminator
+      strlcpy(suffix, " ", sizeof(suffix));
+      strlcat(suffix, coin->coin_shortcut, sizeof(suffix));
```

Comment:
changed from #278: use `strlcat` instead of `snprintf`; slightly lighter, probably slightly safer